src/app/reducers/knowledge.ts

```diff
@@ -0,0 +1,29 @@
+import { compose } from '@ngrx/core';
+import { combineReducers } from '@ngrx/store';
+import { ActionReducer, Action } from '@ngrx/store';
+import { createSelector } from 'reselect';
+import * as knowledge from '../actions/knowledge';
+export const CHANGE: Action = {
```

Comment:
Use already created Action.

Reply:
It is not used in project, so no need to make it more complex. I removed it but coverage was decreasing by `0.03%` so I left it.

Reply:
If its not required so why are you using it? For increasing coverage you can write many more lines of code.